### PR TITLE
removes sus

### PR DESCRIPTION
**Pull request by Skrem7**
# Document the changes in your pull request

I forgor to remove the placeholder description (there were many things I had to keep track of)

# Changelog

:cl:  
spellcheck: CMO plasmaman envirosuit is now significantly less sus
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/jobs/Plasmaman/medsci.dm [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 /obj/item/clothing/under/plasmaman/cmo
 	name = "chief medical officer's envirosuit"
-	desc = "Sus"
+	desc = "A white-blue envirosuit to denote a plasmaman of the high rank of \"Chief Medical OFficer\". Quite comfortable."
```
Comment (tattax) [inline code/modules/clothing/under/jobs/Plasmaman/medsci.dm]:
```suggestion
	desc = "A white-blue envirosuit to denote a plasmaman of the high rank of \"Chief Medical Officer\". Quite comfortable."
```

**Review by tattax — COMMENTED** [review]
hello

**Review by TheRyeGuyWhoWillNowDie — CHANGES_REQUESTED** [review]
close this pr

**Comment by MajManatee** [comment]
gus is right
also add a backup enviro suit to each head locker or somthing idk

**Review by ynot01 — APPROVED** [review]
OFficer

**Pull request merged**